static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -256,126 +274,141 @@ define([
             }
         });
 
-        // Location of this is important.
-        events.bindErrorHandler(player);
-        player.guMediaType = mediaType;
-
-        playerSetupComplete = new Promise(function (resolve) {
-            player.ready(function () {
-                var vol;
-
-                deferToAnalytics(function () {
-                    events.initOmnitureTracking(player);
-                    events.initOphanTracking(player, mediaId);
-
-                    events.bindGlobalEvents(player);
-                    events.bindContentEvents(player);
-                    if (withPreroll) {
-                        events.bindPrerollEvents(player);
-                    }
+        isPlayerExpired.then(function(expired) {
+            if (expired) {
+                player.ready(function() {
+                    player.error({
+                        code: 0,
```

Comment:
No. As we aren't 100% sure why something might be invalid i.e. it's our own logic.